tests/api/test_types.py

```diff
@@ -125,3 +125,16 @@ def test_sas_code():
     for invalid in ["", "AAA", "AAAAA", "aaaa", "AAAI", "AAAO", "AAA0", "AAA1"]:
         with pytest.raises(ValueError):
             SASCode(invalid)
+
+
+def test_entryname_normalization():
+    nfc_str = "àáâäæãåāçćčèéêëēėęîïíīįìłñńôöòóœøōõßśšûüùúūÿžźż"
+    nfd_str = (
```

Comment:
Funny to see black doesn't format the two lines the same way even though there look the same length 😄 

I guess it can be explained by:
```python
>>> len("àáâäæãåāçćčèéêëēėęîïíīįìłñńôöòóœøōõßśšûüùúūÿžźż")
47
>>> len("àáâäæãåāçćčèéêëēėęîïíīįìłñńôöòóœøōõßśšûüùúūÿžźż")
89
```